### PR TITLE
Added Amadeus Pro.app v1.5.5

### PR DESCRIPTION
**Pull request by shimeike**
- legacy version of this app

### CODE REVIEW

**Comment by shimeike** [comment]
Did I break something .... ?

File: Casks/amadeus-pro1x.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'amadeus-pro1x' do
+  version '1.5.5'
+  sha256 '29e5d6a2567c9b1a418667b99844de186a6621992f3398583c959aed2383d0b6'
+
+  # This is the 1.x version of Amadeus Pro
```
Comment (vitorgalvao) [inline Casks/amadeus-pro1x.rb]: Please remove this comment. It’s useless.

**Comment by shimeike** [comment]
Did I push or squash this incorrectly? The cask passes audit and installs locally. Sorry, but I can't make sense of the logs ... (noob!)

**Comment by vitorgalvao** [comment]
Travis was just acting weird, apparently,

**Pull request merged**